### ISSUE TITLE
With no whitespace rendering EOL still visible

### ISSUE THREAD

**Issue by crystalfp** [medo64/Render-CRLF#8]
In the previous version the EOL character was not visible if the "Render whitespace" in the menu is not turned on.
With this version (same settings as before) the EOL character is always visible for markdown files (see below). In the language specific section for markdown I have  `"editor.renderWhitespace": "boundary"`. For other filetypes that have no  `"editor.renderWhitespace"` setting, the extension works as should.

The extension settings are:
```
    "code-eol.style": {
        "color" : "#5b5c55",
        "opacity" : 1.0
    },
    "code-eol.highlightExtraWhitespace": true,
    "code-eol.toggleWithWhiteSpace": true,
```
What should I set to have the EOL character visible only when the menu "Render whitespace" is checked for markdown files? Is the visibility not changed if `renderWhitespace` is set to` boundary`?
Thanks!
mario

**Comment by medo64** [comment]
Can you please check if behavior is as expected now? This was a bug in 1.5.5 - it's fixed in 1.5.6 released today,

**Comment by crystalfp** [comment]
Problem still here.
Also seems to ignore `"code-eol.toggleWithWhiteSpace": true,` setting.
Strange

**Comment by medo64** [comment]
Hi,

"code-eol.toggleWithWhiteSpace": true was never supported - this extension always toggles with Render Whitespace. You might be referring to code-eol 2019 extension.

After giving it some though, I will change 'boundary' to behave as 'none'. Since intention of that setting is to display only boundary whitespace characters, it sort-of made sense to have EOL characters rendered there too. However, if intention of 'boundary' is to minimize whitespace noise, it kinda makes sense to not render EOLs. I went with the later idea.

In any case, I described behavior in readme now so there is no confusion in the future.
